controls/repository.sp

```diff
@@ -41,6 +41,7 @@ benchmark "repository_checks" {
   children = [
     benchmark.repository_mod_checks,
     benchmark.repository_plugin_checks,
+    benchmark.repository_steampipe_cli_fdw_sdk_docs_checks
```

Comment:
```suggestion
    benchmark.repository_steampipe_core_checks
```

Not sure `core` is the right name, but I think it's shorter and still applies to all of the repos in scope